src/module/classes/barbarian.mjs

```
@@ -1,32 +1,22 @@
+    
+    // Set the default notification message
+    let notification = "elkan5e.notifications.FeralInstincts";
+    
+    // Check if the actor has Feral Instincts or Improved Feral Instincts
+    if (RAGE && (actor.items.find(i => i.name === "Feral Instincts") || actor.items.find(i => i.name === "Improved Feral Instincts"))) {
+        if (actor.items.find(i => i.name === "Improved Feral Instincts")) {
+            notification = "elkan5e.notifications.ImprovedFeralInstincts";
+        }
         
-        // console.log("Nothing ")
+        let uses = RAGE.system.uses.max - RAGE.system.uses.spent;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'uses' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)